core-httpclient-impl/src/main/java/com/optimizely/ab/config/HttpProjectConfigManager.java

```diff
@@ -125,14 +130,30 @@ protected ProjectConfig poll() {
         return null;
     }
 
+    HttpGet createHttpRequest() {
+        HttpGet httpGet = new HttpGet(uri);
+
+        if (authDatafileToken != null) {
+            httpGet.setHeader(HttpHeaders.AUTHORIZATION, "Bearer " + authDatafileToken);
+        }
+
+        if (datafileLastModified != null) {
+            httpGet.setHeader(HttpHeaders.IF_MODIFIED_SINCE, datafileLastModified);
+        }
+
+        return httpGet;
+    }
+
     public static Builder builder() {
         return new Builder();
     }
 
     public static class Builder {
         private String datafile;
         private String url;
+        private String authDatafileToken = null;
         private String format = "https://cdn.optimizely.com/datafiles/%s.json";
+        private String authFormat = "https://www.optimizely-cdn.com/datafiles/auth/%s.json";
```

Comment:
The domain is `config.optimizely.com`:
```suggestion
        private String authFormat = "https://config.optimizely.com/datafiles/auth/%s.json";
```